README.md

```diff
@@ -47,6 +47,23 @@ await formEditor.importSchema(schema);
 See [editor documentation](./packages/form-js-editor) for further details.
 
 
+### Create and Simulate a Form with Input and Output Data <a id="playground" />
```

Comment:
```suggestion
### Create and simulate a form with input and output data
```

Let's follow the Camunda Writing Style Guide, "Titles and headers" section: https://camunda.com/brand/writing-style-guide/#CamundaWritingStyleGuide-Formatting,organization,andstructureforconceptualpieces

Reply:
Thanks, I introduced sentence case via https://github.com/bpmn-io/form-js/pull/320/commits/66891836f8226e05d8cffe97776313988de8e4ec!